tutorial/accesstype-integration.md

```diff
@@ -0,0 +1,610 @@
+---
+title: Integrating Accesstype with Malibu Advanced
+nav_order: 32
+parent: Malibu Tutorial
+---
+
+# {{page.title}}
+
+_This tutorial was contributed by [Phaneendra Andukuri](https://www.linkedin.com/in/venkata-phaneendra-andukuri/)_
+
+In this document we will understand how to integrate accesstype with malibu-advanced apps. Accesstype is a subscription and access management platform built by quintype.
+
+## Accesstype Account Setup:
+
+1. Create accesstype account: Go to <https://www.accesstype.com/dashboard> for prod instance or (<https://staging.accesstype.com> for staging) and login.
+1. In the accesstype dashboard, go to settings > general, scroll down to see Account key inside Authentication Keys category. Keep this key handy for further use
+1. Talk to quintype supports to create bridgekeeper-accesstype integration for your domain. They will give you an integration Id which we will use later.
+1. Create a razorpay account and get api_key, secret from the razorpay dashboard and add them in accesstype dashboard > configure > payment gateways > razorpay and click save
+   **Note**: Accesstype support other payment gateways also but for now We have integrated only razorpay payment gateway in malibu-advanced
+1. Now that we have created an accesstype account and integrated it with razorpay payment gateway, we will now create some subscription groups in the dashboard.
+1. Go to dashboard > group_and_plans > manage click on add subscriptions and add name, description and other details and click save.
+1. Now you can configure other options like coupon codes in the dashboard > configure tab. Click on add coupon and add the name and discount value and click save.
+
+Now that we are done with the accesstype account setup, let's add the access type config in the blackKnight config. For that, go to your blackKnight account > Config Files, in that edit your publisher.yml file and add the below snippet replacing your-accesstype-account-key bk-accesstype-integration-id with the actual values.
+
+```yml "
+accesstypeConfig:
+  key: "<your-accesstype-account-key>"
+  accessTypeBkIntegrationId: "<bk-accesstype-integration-id>"
+```
+
+We have all the setup done. We can now add the functionality and UI through our code but before that see this outline of what all we are going to do.
+
+1. Creating a subscriptions page
+1. Showing all the subscription groups
+1. Showing Login modal with selected subscription plan for anonymous users
+1. Showing Checkout modal for the logged in users
+1. Making payment through Razorpay
+1. Adding subscriptions to the user profile
+1. Adding a paywall to the stories that cannot be accessible to the user
+1. Enabling subscription and metering paywall features for amp stories
+
```

Comment:
Numbers seems repeated

Reply:
it seems to be coming like but when I look in markdown viewer it is shown properly